packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.types.ts

```diff
@@ -246,6 +246,15 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox,
    * Optional iconButton props on combo box
    */
   iconButtonProps?: IButtonProps;
+
+  /**
+   * Custom render function for the label text.
+   */
+  onRenderLabel?: (
+    props?: IComboBoxProps,
+    multiselectAccessibleText?: string,

```

Comment:
I think put this into a single prop object rather than having IComboBoxProps and muktiselectAccessibleText be separate. We might want to keep things consistent with the [IRenderFunction](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/utilities/src/IRenderFunction.ts).

Reply:
If we could use IRenderFunction that'd keep the interface consistent with other `onRender` functions.